### DISCUSSION TITLE
Using GetOrganelle for a bacterial chromosome

### DISCUSSION THREAD

**Discussion by liamfriar** [Kinggerm/GetOrganelle#249] [Q&A] [answered]
I am aware that it is not the intended use of GetOrganelle, but I am curious if people think that GetOrganelle could be used to assemble a small (~5Mb) bacterial chromosome? There is a complete sequence from a related strain to use as a reference, or I have some contigs from a SPAdes assembly that was unable to fully close the chromosome.

**Comment by JianjunJin** [comment, marked as answer]
GetOrganelle will only be beneficial when the target chromosome has a significantly higher depth than the others in the bulk sequences pool. I'm not sure if it is your case.

**Reply by liamfriar** [reply]
Thank you for your answer! Just to make sure I understand: if the reads are mostly from the target chromosome, then GetOrganelle might work? Would you give it an assembly with the contigs connected into one long sequence by runs of Ns as the seed? Or just give one contig as a seed? Or all the contigs separately? Thank you again!

**Reply by JianjunJin** [reply]
I need to rephrase it. GetOrganelle will only be beneficial when the target chromosome has a significantly higher depth (>20 folds; e.g. pt 200x vs nucl 10x) and a relatively low percentage (e.g. <10%) than the remaining contents in the bulk sequences pool. 

So if the reads are mostly from the target, please just use SPAdes or other general assemblers. Please refer to the [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02154-5) for more info.

**Reply by liamfriar** [reply]
Okay. Thank you, Dr. Jin!
